src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -0,0 +1,114 @@
+Make sure you have completed the below steps:
+
+- [Completed the pre-requisite steps.](/lib/in-app-messaging/prerequisites)
+- [Create a campaign from the pinpoint console.](/lib/in-app-messaging/create-campaigns)
+
+## Integrate your application
+
+In your application directory, you should first Install necessary dependencies.Although Amplify In-App Messaging can be used as a standalone JavaScript library, the only UI integration currently available is only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
+
+```js
+npm install aws-amplify@in-app-messaging &&
+npm install aws-amplify-react-native@in-app-messaging &&
+npm install amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
```

Comment:
We should be able to combine this
```suggestion
npm install aws-amplify@in-app-messaging aws-amplify-react-native@in-app-messaging amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
```

Reply:
Also I think we may need a couple more polyfills:
```sh
react-native-get-random-values
react-native-url-polyfill
```
Maybe @calebpollman can chime in on this one. They either need to be here or included as peer dependencies of the RN package 🤔